docs/sql-ref-syntax-ddl-create-database.md

```diff
@@ -19,4 +19,39 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Creates a database with the given name. If database with the same name exists an exception will be thrown.
+
+### Syntax
+{% highlight sql %}
+CREATE (DATABASE|SCHEMA) [IF NOT EXISTS] database_name
+  [COMMENT database_comment]
+  [LOCATION database_directory]
+  [WITH DBPROPERTIES (property_name=property_value, ...)]
+
+{% endhighlight %}
+
+### Parameters
+- ##### ***IF NOT EXISTS***:
+Creates a database with the given name if it doesn't exists. If a database with the same name already exists, nothing will happen.
+- ##### ***LOCATION***:
```

Comment:
Since this is an optional clause, should we specify what happens when this is not specified ? The database is created in the default warehouse directory ? Could you please check ?

Reply:
Updated. Thanks